lib/Service/ConfigureCheckService.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace OCA\Libresign\Service;
+
+use ImagickException;
+use OC\SystemConfig;
+use OCA\Libresign\AppInfo\Application;
+use OCA\Libresign\Handler\JSignPdfHandler;
+use OCP\IConfig;
+
+class ConfigureCheckService {
+	private IConfig $config;
+	private SystemConfig $systemConfig;
+
+	public function __construct(
+		IConfig $config,
+		SystemConfig $systemConfig
+	) {
+		$this->config = $config;
+		$this->systemConfig = $systemConfig;
+	}
+
+	public function canPreview(): array {
+		if (!extension_loaded('imagick')) {
+			return [
+				'errors' => ['Extension Imagick required'],
+			];
+		}
+
+		$imagick = new \Imagick();
+		$imagick->setResolution(100, 100);
+		$pdf = file_get_contents(__DIR__ . '/../../tests/fixtures/small_valid.pdf');
+		try {
+			$imagick->readImageBlob($pdf);
+		} catch (ImagickException $ie) {
+			if ($ie->getCode() === 499) {
+				return [
+					'errors' => ['Necessary configure the security policy of ImageMagick to work with PDF. More informations: https://github.com/LibreSign/libresign/issues/829'],
```

Comment:
Is necessary to configure the ImageMagick security policy to work with PDF. More information on https://github.com/LibreSign/libresign/issues/829'